cves/CVE-2017-11665.yml

```diff
@@ -130,7 +130,16 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
```

Comment:
add an upvote, I thought this vulnerability was pretty interesting

Reply:
Upvote added